### DISCUSSION TITLE
Les monarques

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#36] [Q&A]
Commentaires sur ce quiz

**Comment by ArtificialOxygen** [comment]
Écrit par Lorie-Anne Côté
# Petit retour sur ton quiz "Les monarque de l'histoire"

## Points forts
- Très agréable visuellement: un design très bien choisi qui nous rappelle vraiment le sujet de l'histoire "ancienne".
- Compréhension aisée: on comprend bien comment naviguer au travers de ton quiz. On sait aussi quelle réponse on coche parce que tu lui as donné un état différent (avec un contour mauve qui est d'ailleurs un bon contraste avec les couleurs de l'arrière plan). 
- Sujet très intéressant ! J'ai beaucoup appris avec ton quiz.

## Points à améliorer 
- Quelques petites fautes de grammaire, mais elles ne nuisent pas à la compréhension du quiz ni à la lecture des textes.
- Peut-être que c'est moi le problème, mais je crois qu'il n'est pas possible de naviguer sur tes boutons radio en utilisant la touche TAB ainsi que les flèches.
- Attention, je crois que tu as peut-être oublié de changer le contenu de la balise title dans le head du document HTML (présentement nommé "Titre du quiz - Quiz TIM").

**Important: j'ai écrit ceci ne sachant pas si tu avais terminé ton quiz. Donc, peut-être que tu as encore quelques modifications à apporter comme ce que j'ai nommé plus haut dans "points à améliorer".**

**Reply by phoenix-four** [reply]
Hey merci de tes commentaires! :D

**Comment by kateundercoffler** [comment]
Vraiment intéressant ton sujet ! J'aime bien la palette de couleur, ça à l'air de quelque chose de l'antiquité.  J'ai aime le transition des boutons aussi en survol.  
Pour améliorer : Je ne suis pas certaine si c'est juste moi, mais j'ai eu la difficulté en utilisant le touche clavier pour naviguer les choix de réponses.  Est-ce que c'est normal ?  Je n'ai pas pu transition entre un choix ou un autre avec tab + mes flèches gauche ou droit.  Aussi, attention avec le contrast pour la police sur les boutons, peut-être tu peux mettre en gras les textes ? 

Bravo !! 👍

**Reply by phoenix-four** [reply]
En effet, je ne sais pas pourquoi la navigation clavier ne marche pas ^^'

**Comment by evefevrier** [comment]
@phoenix-four 
Vraiment TRÈS intéressant comme quiz! Merci de nous faire découvrir ce sujet.
Pour l'orthographe: est-ce que le titre ne devrait pas avoir un "s" ? et puis à la rétro de la dernière question il y a "Chléopâtre".
